client/www/pages/dash/index.tsx

```diff
@@ -135,20 +174,31 @@ function isTabAvailable(tab: Tab, role?: Role) {
   return tab.minRole ? role && isMinRole(tab.minRole, role) : true;
 }
 
-function Dashboard() {
+function Dashboard({ onClaimTicket }: { onClaimTicket: () => void }) {
   const token = useContext(TokenContext);
   const router = useRouter();
   const appId = router.query.app as string;
   const screen = (router.query.s as string) || 'main';
   const _tab = router.query.t as TabId;
   const tab = tabIndex.has(_tab) ? _tab : defaultTab;
+  const ticket = router.query.ticket as string | undefined;
 
   const [connection, setConnection] = useState<{
     db: InstantReactClient;
   } | null>(null);
 
   const dashResponse = useAuthedFetch<DashResponse>(`${config.apiURI}/dash`);
 
+  // new CLI login flow
+  useEffect(() => {
+    if (!token) return;
+    if (!ticket) return;
+    claimTicket({ ticket, token }).then(() => {
+      onClaimTicket();
```

Comment:
Should we handle the failure case? For example, if the user starts the auth process, but closes their cli, in that case, this 'ticket' would fail?